### PR TITLE
1.40 Changes

### PR DESCRIPTION
**Pull request by murrant**

### CODE REVIEW

File: notifications.rss [hunk 1]
```diff
@@ -148,5 +148,10 @@
   <description>On April 12th we will merge a pull request which will rename the OS' avaya-ers and avaya-vsp OS' to boss and voss respectively.  Please adjust any alert rules or device groups that use these OS names.</description>
   <pubDate>Sat, 07 Apr 2018 20:00:00 +0000</pubDate>
  </item>
+  <item>
+  <title>Manual intervention required for 1.40 update</title>
+  <description>When LibreNMS 1.40 is released on Sunday May 27, there are some steps that need to be taken to restore your Web UI.  The poller should countinue without interruption.  More details here: https://community.librenms.org/t/new-requirements-for-1-40/4057</description>
```
Comment (laf) [inline notifications.rss]: Change to Wed 30th?

Comment (SaaldjorMike) [reply notifications.rss]: Also, `s/countinue/continue/` 😄

Comment (murrant) [reply notifications.rss]: Wednesday? for more time?

**Comment by murrant** [comment]
@laf Wednesday? for more time?

**Comment by laf** [comment]
@murrant Yes, or the sunday after. I'm not comfortable with just giving people 4-5 days notice.

**Comment by murrant** [comment]
I like the middle of the week better.

**Pull request merged**